pygac/pod_reader.py

```diff
@@ -315,7 +315,6 @@ def _truncate_padding_record(self, buffer):
         containing only one scanline and one "logical" record of padding.
         """
         scanlines_per_record = self.offset // self.scanline_type.itemsize
-        assert scanlines_per_record in [1,2], f"POD files should have 1 or 2 logical records per physical, not {scanlines_per_record}"
```

Comment:
Oh, I didn’t mean for you to remove this error, I think it make sense actually, but rather have something like:
```python
if not scanlines_per_record in [1,2]:
    raise IOError(f"POD files should have 1 or 2 logical records per physical, not {scanlines_per_record}")